phenol-io/src/test/java/org/monarchinitiative/phenol/io/obo/hpo/HpoDiseaseAnnotationParserTest.java

```diff
@@ -1,60 +1,130 @@
 package org.monarchinitiative.phenol.io.obo.hpo;
 
-import com.google.common.collect.ImmutableCollection;
-import com.google.common.collect.ImmutableMultimap;
-import org.junit.Before;
-import org.junit.Rule;
+
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
 import org.monarchinitiative.phenol.base.PhenolException;
+import org.monarchinitiative.phenol.formats.hpo.HpoAnnotation;
 import org.monarchinitiative.phenol.formats.hpo.HpoDisease;
 import org.monarchinitiative.phenol.formats.hpo.HpoOntology;
 import org.monarchinitiative.phenol.io.utils.ResourceUtils;
 import java.io.File;
 import java.io.IOException;
+import java.util.List;
 import java.util.Map;
-import java.util.Optional;
+import java.util.stream.Collectors;
 
 import org.junit.rules.TemporaryFolder;
 import org.monarchinitiative.phenol.ontology.data.TermId;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
+/**
+ * These tests were constructed with a mini hpo file and a mini annotation file -- look at them to
+ * figure out the logic!
+ */
 public class HpoDiseaseAnnotationParserTest {
-  private HpoDiseaseAnnotationParser parser;
-  private Map<TermId, HpoDisease> parsedFile;
+  private static Map<TermId, HpoDisease> diseaseMap;
 
-  @Rule
-  public TemporaryFolder tmpFolder = new TemporaryFolder();
+  @ClassRule
+  public static TemporaryFolder tmpFolder = new TemporaryFolder();
 
-  @Before
-  public void setUp() throws IOException, PhenolException {
+  @BeforeClass
+  public static void setUp() throws IOException, PhenolException {
     System.setProperty("user.timezone", "UTC"); // Somehow setting in pom.xml does not work :(
     File hpoHeadFile;
 
 
     hpoHeadFile = tmpFolder.newFile("hp_head.obo");
     ResourceUtils.copyResourceToFile("/hp_head.obo", hpoHeadFile);
-    final HpOboParser oboParser = new HpOboParser(hpoHeadFile, true);
+    final HpoOboParser oboParser = new HpoOboParser(hpoHeadFile, true);
```

Comment:
@pnrobinson  This does not pass a mvn clean test because the `HpoOboParser` doesn't exist.